example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -961,6 +963,8 @@ describe('When there are granule differences and granule reconciliation is run',
     console.log(`update database state back for  ${publishedGranuleId}, ${activeCollectionId}`);
     await updateGranule({
       prefix: config.stackName,
+      granuleId: publishedGranuleId,
+      collectionId: granuleBeforeUpdate.collectionId,
```

Comment:
These changes were necessary due to https://github.com/nasa/cumulus/pull/3022/files#diff-4cc682d0b21198b6eebd2261cc680c5a7799cc3a1a961eb3b1969e39e27483dcR516

While I was at it I also added the `collectionId` param.